v3.0-RC2/system_alerts.json

```
@@ -0,0 +1,165 @@
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "region_ids": {
+                "description":
+                  "Array of identifiers of the regions for which this alert applies.",
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "url": {
+                "description":
+                  "URL where the customer can learn more information about this alert.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "text": {
+                      "description": "The translated text.",
+                      "type": "string",
+                      "format": "uri"
+                    },
+                    "language": {
+                      "description": "IETF BCP 47 language code.",
+                      "type": "string",
+                      "pattern": "^[a-z]{2,3}(-[A-Z]{2})?$"
+                    }
+                  },
+                  "required": ["text", "language"]
+                }
+              },
+              "summary": {
+                "description":
+                  "A short summary of this alert to be displayed to the customer.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "text": {
+                      "description": "The translated text.",
+                      "type": "string"
+                    },
+                    "language": {
+                      "description": "IETF BCP 47 language code.",
+                      "type": "string",
+                      "pattern": "^[a-z]{2,3}(-[A-Z]{2})?$"
+                    }
+                  },
+                  "required": ["text", "language"]
+                }
+              },
+              "description": {
+                "description": "Detailed description of the alert.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "text": {
+                      "description": "The translated text.",
+                      "type": "string"
+                    },
+                    "language": {
+                      "description": "IETF BCP 47 language code.",
+                      "type": "string",
+                      "pattern": "^[a-z]{2,3}(-[A-Z]{2})?$"
+                    }
+                  },
+                  "required": ["text", "language"]
+                }
+              },
+              "last_updated": {
+                "description":
+                  "Indicates the last time the info for the alert was updated.",
+                "type": "number",
+                "minimum": 1450155600
```

Let's please update the type and format to [RFC3339 format](https://www.rfc-editor.org/rfc/rfc3339).

Thank you!

```suggestion
                "type": "string",
                "format": "date-time"
```